Pod/Classes/LSPAudioViewController.m

```diff
@@ -151,7 +151,6 @@ - (void)play
 {
     DELEGATE_SAFELY(self.delegate, @selector(audioViewController:willPlayPlayer:), [self.delegate audioViewController:self willPlayPlayer:self.player];)
     
-    [self.audioQueuePlayer removeTimeObserver:_progressObserver];
```

Comment:
Why blindly remove this? What if a time observer already exists? Should you at least check to see if one exists and remove it if necessary?